### DISCUSSION TITLE
Can I use pip command to install the packages locked in pdm.lock?

### DISCUSSION THREAD

**Discussion by NikelChamp** [pdm-project/pdm#3088] [Q&A] [answered]
When I want to deploy my project, the new environment has no pdm on it. How can I use the pip command to install packages? Using requirements.txt instead? And I think it would take a long time to install pdm If I build a mirror. How would you guys solve it?

**Comment by frostming** [comment, marked as answer]
`pdm export -o requirements.txt`

**Reply by NikelChamp** [reply]
Whoo! Thanks pal!
